portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,30 +27,50 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+
+  List<String> comments = new ArrayList<String>();
   private final Gson gson = new Gson();
 
   @Override
   public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
-        
-        // Initialise messages with three hard-coded values
-        List<String> messages = new ArrayList<String>();
-        messages.add("Lesser leather never weathered wetter weather better");
-        messages.add("Which witch is which?");
-        messages.add("Fred fed Ted bread, and Ted fed Fred bread");
-
-        // Convert to JSON format
-        String json = toJsonArray(messages);
-
-        // Send JSON as response
-        response.setContentType("application/json;");
-        response.getWriter().println(json);
-    }
 
-    /**
-     * Converts input to a JSON Array
-     */
-    private String toJsonArray(List<String> data) {
-        final String json = gson.toJson(data);
-        return json;
+    // Convert to JSON format
+    String json = toJsonArray(comments);
+
+    // Send JSON as response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+  * Converts input to a JSON Array
```

Comment:
Can you add @param and @return javadoc tags to this, sorry forgot to pick this up last review

Reply:
Done in the "added code review changes" commit